BnbnavNetClient/App.axaml

```diff
@@ -39,6 +39,15 @@
     <Style Selector="TextBlock">
       <Setter Property="Margin" Value="2"/>
     </Style>
+    <Style Selector="Button.destructive">
+      <Setter Property="Background" Value="#ffff0000" />
```

Comment:
for consistency with the road brushes it'd probably be better not to include the alpha (ff at the start)